cql3/restrictions/statement_restrictions.cc

```diff
@@ -1021,5 +1022,42 @@ ::shared_ptr<single_column_restriction> single_column_restriction::LIKE::apply_t
     return r;
 }
 
+namespace wip {
+
+::shared_ptr<expression> make_conjunction(::shared_ptr<expression> a, ::shared_ptr<expression> b) {
+    if (!a) {
+        return b;
+    }
+    if (!b) {
+        return a;
+    }
```

Comment:
Perhaps we should decree that expressions are never null. It means that a missing WHERE clause gets defaulted to a shared_ptr<true_constant>. It will simplify all the code that deals with expressions.

Reply:
Doesn't apply now that expressions are straight-up values.